src/styles/menu.scss

```diff
@@ -0,0 +1,198 @@
+$padding-mobile: 27px;
+
+.menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 10;
+  background-color: rgba(24, 24, 26, 0.5);
+  transition: opacity 0.3s ease-in-out;
+  opacity: 0;
+  pointer-events: none;
+
+  &:target {
+    opacity: 1;
+    pointer-events: all;
+    & :first-child {
+      animation: 0.3s ease-in-out 0s 1 movetop;
+
+      @include on-tablet {
+        animation: 0.3s ease-in-out 0s 1 movetop-tablet;
+      }
+
+      @include on-desktop {
+        animation: 0.3s ease-in-out 0s 1 movetop-desktop;
+      }
+    }
+  }
+
+  & :first-child {
+    animation: 0.3s ease-in-out 0s 1 movedown;
+
+    @include on-tablet {
+      animation: 0.3s ease-in-out 0s 1 movedown-tablet;
+    }
+
+    @include on-desktop {
+      animation: 0.3s ease-in-out 0s 1 movedown-desktop;
+    }
+  }
```

Comment:
You have a lot of repeated code for animations, consider creating a mixin to avoid code repetition.